### ISSUE TITLE
UX issue with numeric fields

### ISSUE THREAD

**Issue by mccrispy** [SteveBallantine/spirit-island-randomiser#16]
From a UX PoV I think that +/- spinner buttons (or whatever they are called) would improve the ease of use of the various numeric fields: it's easier to click on a +/- button than select a field and then overtype a number.

**Comment by SteveBallantine** [comment]
I think they're a bit of a marmite thing. Personally I hate those spinner buttons and never use them!
However, it's not a big deal to add them.
Interestingly, there are up/down arrows that will appear in those fields when you view the website on desktop. I think they're just a bit small to be practical on a touch screen, so will need to find an alternative.

**Comment by mccrispy** [comment]
Yes, I'm really mainly interested in this as an enhancement for mobile.
